presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -479,10 +479,8 @@ public Map<SchemaTableName, List<ColumnMetadata>> listTableColumns(ConnectorSess
     public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle columnHandle)
     {
         IcebergColumnHandle column = (IcebergColumnHandle) columnHandle;
-        return ColumnMetadata.builder()
-                .setName(column.getName())
-                .setType(column.getType())
-                .setComment(column.getComment())
+        return ColumnMetadata.builder(column.getName(), column.getType())
+                .setComment(column.getComment().orElse(null))
```

Comment:
Wondering if it is worth adding a `setComment` with an `Optional` argument. I haven't really counted if there is more than this use, but I could see it being useful